src/sentry/spans/consumers/process/factory.py

```diff
@@ -37,6 +37,19 @@
 BATCH_SIZE = 100
 
 
+def in_process_spans_rollout_group(project_id: int | None) -> bool:
+    if project_id and project_id in options.get(
+        "standalone-spans.process-spans-consumer.project-allowlist"
+    ):
+        return True
+
+    if project_id and (project_id % 100000) / 100000 < options.get(
+        "standalone-spans.process-spans-consumer.project-rollout"
+    ):
+        return True
+    return False
```

Comment:
You could use the `in_rollout_group` helper function here. But it hashes the project ID, not sure if that's what you want.